### DISCUSSION TITLE
Keep scrolling, memory usage will keep increasing

### DISCUSSION THREAD

**Discussion by ShangChien** [rocwang/vue-virtual-scroll-grid#238] [Q&A]
![image](https://user-images.githubusercontent.com/48350636/179456786-0923958a-201b-466f-a513-489d7fa42e03.png)

When embedding a slightly more complex component, the memory usage rise is more pronounced? If the size of each item is the same, the virtual scroll will exceed the pagesize. Memory should not increase any more.
thank you.

**Comment by rocwang** [comment]
The `vue-virtual-scroll-grid` component caches all loaded items from `pageProvider` at the moment, which might lead to the rise of memory usage. A new prop to toggle the caching function might help here.

The complex `rxjs` streams under the hood might also contribute to the issue.

I'll see if I can profile it and find the culprit.
